packages/commons/src/queue/sqs.ts

```diff
@@ -116,14 +117,17 @@ export const sendMessage = async (
 export const deleteMessage = async (
   sqsClient: SQSClient,
   queueUrl: string,
-  receiptHandle: string,
+  messagePayload: Message,
   loggerInstance: Logger
 ): Promise<void> => {
+  const { ReceiptHandle, Body } = messagePayload;
+  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+  const correlationId = JSON.parse(Body!).correlationId as unknown;
```

Comment:
IMHO it is acceptable a weak typing to obtain CID to simplify debug and troubleshooting.
At this point, after checking and saving the message, message body has all values, including correlationId